qa/suites/rados/thrash-old-clients/1-install/hammer.yaml

```diff
@@ -1,41 +0,0 @@
-overrides:
-  ceph:
-    mon_bind_msgr2: false
-    crush_tunables: hammer
-    log-whitelist:
-      - \(MON_DOWN\)
-    conf:
-      global:
-        ms bind msgr2: false
-      mon:
-        mon osd initial require min compat client: hammer
-tasks:
-- install:
-    branch: hammer
```

Comment:
The thrash-old-clients hammer tests seem to have started failing recently on octopus because of missing packages https://tracker.ceph.com/issues/47915, @djgalloway @yuriw do you know what changed?

@jdurgin are we ready to drop hammer client testing? I know we discussed this earlier but decided to drop it for master not for octopus for some reason.

Reply:
> The thrash-old-clients hammer tests seem to have started failing recently on octopus because of missing packages https://tracker.ceph.com/issues/47915, @djgalloway @yuriw do you know what changed?

`2020-10-01T21:00:51.973 DEBUG:teuthology.packaging:Querying https://shaman.ceph.com/api/search?status=ready&project=ceph&flavor=default&distros=ubuntu%2F18.04%2Fx86_64&ref=hammer`

This tried to find Ubuntu 18.04 packages for hammer.  Ubuntu 18.04 didn't even exist when hammer went EOL.  This isn't an infra issue.  Something is misconfigured.

Reply:
I think https://github.com/ceph/ceph/pull/36931 caused it, guess we've been building hammer packages for centos 7.6. Maybe it's time to drop hammer client testing or not use cephadm for thrash-old-clients in octopus.

Reply:
I don’t think that’s necessary. Let’s drop hammer.